src/pages/indregister/indregister.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { IonicPage, NavController} from 'ionic-angular';
+import { MainPage } from '../pages';
+
+
+@IonicPage()
+@Component({
+  selector: 'page-indregister',
+  templateUrl: 'indregister.html'
+})
+export class IndregisterPage {
+ 
+  constructor(public navCtrl: NavController) { }
+  // Our translated text strings 
+  
+
+  
+  login() {
+  
+      this.navCtrl.push('WelcomePage');
```

Comment:
you need to import and push to the resource page. when you push to the welcome, that is where you hosted the login information, in my opinion we should have this redirect into the app as opposed to back out to the login page where they need to login.